src/main/java/com/amihaiemil/docker/ListedNetworks.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2018, Mihai Emil Andronache
+ * All rights reserved.
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 1)Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ * 2)Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * 3)Neither the name of docker-java-api nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.amihaiemil.docker;
+
+import java.net.URI;
+import java.util.Iterator;
+import org.apache.http.client.HttpClient;
+
+/**
+ * Listed networks.
+ * @author Boris Kuzmic (boris.kuzmic@gmail.com)
+ * @since 0.0.7
+ * @todo #211:30min Implement ListedNetworks iterator with filtering using Map.
+ *  See ListedImages or ListedVolumes as examples.
+ */
+final class ListedNetworks extends RtNetworks {
+
+    /**
+     * Ctor.
+     *
+     * @param client The http client.
+     * @param uri The URI for this Network API.
+     * @param dkr The docker entry point.
+     */
+    ListedNetworks(final HttpClient client, final URI uri, final Docker dkr) {
+        super(client, uri, dkr);
+    }
+
+    @Override
+    public Iterator<Network> iterator() {
+        throw new UnsupportedOperationException(
```

Comment:
@bkuzmic I've have an opinion about these tests: don't you think it's better to create the real test for the method and just mark it with `@Ignore` instead making a test that tests some unfinished method? Looks like added tests don't test anything, thay are here just to satisfy coverage metrics

Reply:
@paulodamaso I agree with you that these tests serve just to satisfy coverage metrics. But, I want to minimize my work (since this is a 30min ticket) and I want to force the one who will implement these methods to refactor the unit tests. If I put `@Ignore`, I will not be able to do that.

Reply:
@paulodamaso If a developer changes the implementation in one of the methods in RtNetwork class, the unit test will fail, which means that it has to be refactored in order to pass. But ok, I'll add real test cases and mark them ignored.